src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java

```diff
@@ -31,36 +29,33 @@
 import java.nio.file.Paths;
 import java.time.Duration;
 import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
-import org.junit.platform.commons.support.AnnotationSupport;
-import org.junit.platform.engine.ExecutionRequest;
-import org.junit.platform.engine.TestExecutionResult;
-import org.junit.platform.engine.reporting.ReportEntry;
+import org.junit.jupiter.api.extension.ExtensionContext;
 
 class JazzerFuzzTestExecutor {
-  private static final AtomicBoolean hasExecutedOnce = new AtomicBoolean();
+  private static final AtomicBoolean hasBeenPrepared = new AtomicBoolean();
 
-  private final ExecutionRequest request;
-  private final JazzerFuzzTestDescriptor fuzzTestDescriptor;
-  private final Path baseDir;
+  private final List<String> libFuzzerArgs;
 
-  public JazzerFuzzTestExecutor(
-      ExecutionRequest request, JazzerFuzzTestDescriptor fuzzTestDescriptor, Path baseDir) {
-    this.request = request;
-    this.fuzzTestDescriptor = fuzzTestDescriptor;
-    this.baseDir = baseDir;
+  private JazzerFuzzTestExecutor(List<String> libFuzzerArgs) {
```

Comment:
This is the only class in the JUnit integration prefixed with `Jazzer`. Remove that?

Reply:
Renamed.